src/main/js/helpers/axis.js

```diff
@@ -189,22 +190,38 @@ const getAxesScale = (axis, scale, config) => {
         scale.x,
         getAxisInfoOrientation(config.axis.x.orientation)
     );
+
+    if (
+        (config.ticksCount === undefined ||
```

Comment:
Can you add a comment here that we are checking? What will happen if the consumer doesn't provide Y2 axis

Reply:
I added the comment [here](Added). If the consumer does not provide the Y2 axis, the Y axis ticks will be calculated automatically.